src/collectionExporter.ts

```diff
@@ -231,6 +232,9 @@ class CollectionExporter implements IExporter {
 			duplicateProcessed.add(lrKey);
 		}
 
+		// Ensure items are inserted in correct order
```

Comment:
I see now what is happening here, you reversed the way the toInsert is being populated since you reversed the `loadedItems` loop, but why do you need that if you anyways are just re-reversing the `toInsert` array again.
Also note that we anyways sort the toInsert array `if (await this.sortbyIfLinked(toInsert)) {`

Reply:
Correct, but I am concerned that this changes the order.  Currently is is FIFO but with this change it becomes FILO

Reply:
Yes that sounds good :)